### ISSUE TITLE
[Accessibility] Screen reader is not narrating name for 'Target in page' button present in 'Selenium IDE'

### ISSUE THREAD

**Issue by Winog** [SeleniumHQ/selenium-ide#1306]
## 🐛 Bug Report

Users who rely on screen readers are missing information as Screen reader is not narrating name for 'Target in page' control present in 'Selenium IDE'.
The 4 buttons in the orange rectangle are facing this issue
![image](https://user-images.githubusercontent.com/96061237/146776301-812d4f9c-5af3-42e8-abd6-0c301a26f9f1.png)

## To Reproduce
Have a JAWS or windows Narrator turned-on
1) launch Selenium Ide
2) start a new recording (any website)
3) stop the recording 
4) add a new command "check"
5) navigate using keyboard to "Target in page" and listen

## Expected behavior

Screen reader should narrate name for 'Target in page' control present in 'Selenium IDE'. When focus receives on the control, JAWS should narrate as 'Target in page button'

## Environment

OS: Windows 10
Selenium IDE Version: 3.17.0

Browser: Edge
Browser Version: 96.0.1054.62 (Official build) (64-bit)
